Makefile

```diff
@@ -344,18 +344,16 @@ olcesium: .build-artefacts/ol-cesium
 	cd .build-artefacts/ol-cesium; \
 	git reset HEAD --hard; \
 	git fetch --all; \
-	git checkout $(OL_CESIUM_VERSION); \
 	git show; \
-	git submodule update --recursive --init --force; \
 	cd ol; \
 	git reset HEAD --hard; \
 	git fetch --all; \
-	git checkout $(OL_VERSION); \
 	git show; \
 	npm install --only prod; \
 	node tasks/build-ext.js; \
 	cd ../cesium; \
 	git remote | grep c2c || git remote add c2c git://github.com/camptocamp/cesium; \
+	git remote | grep oterral || git remote add oterral git://github.com/oterral/cesium; \
```

Comment:
Is that just a convenient way to test thing directly from your repo or do we use it?

Reply:
It will be removed when the PR will be ready

Reply:
yes it for testing branch on my fork